### PR TITLE
Add button cart

### LINKED ISSUE #60: Add Shopping Cart Button to the Header
## 📘 Description

A shopping cart button needs to be added to the header for quick access to the cart and easier product review before checkout.

## 🔢 Steps

1. Add the shopping cart button to the header, ensuring it aligns with the other elements.
2. Ensure the button is responsive and looks good.

## ✅ Acceptance Criteria

- The button is correctly aligned with the other header elements, and its responsive design.
- Use the icons from Lucide React.

## 🌎 References

- [Lucide React](https://lucide.dev/icons/).

## 📜 Additional Notes

- Blocked by: #54

⚠ **Do not apply until the ODHack begins.**
⚠ **Read our guidelines before applying.**

### PR DESCRIPTION
**Pull request by SimplementeCao**
# 📝 Pull Request Title

## 🛠️ Issue
- Closes #60

## 📖 Description
- A shopping cart button needs to be added to the header for quick access to the cart and easier product review before checkout.

## ✅ Changes made
- Add the corresponding button and make sure it is responsive.

## 🖼️ Media (screenshots/videos)
-
Screenshot from 2024-12-12 21-56-56

Screenshot from 2024-12-12 21-56-33

## 📜 Additional Notes
-

### CODE REVIEW

File: frontend/app/marketplace/page.tsx [hunk 1]
```diff
@@ -312,10 +313,17 @@ function HeaderComponent({ searchTerm, setSearchTerm }: HeaderComponentProps) {
 					<span className="sr-only">Search</span>
 				</Button>
 			</div>
-			<Button size="lg" className="group">
-				<Wallet className="mr-2 h-4 w-4 transition-transform group-hover:scale-110" />
-				Connect Wallet
-			</Button>
+
+			<div className="flex items-center gap-2">
+				<Button size="lg" className="group">
```
Comment (derianrddev) [inline frontend/app/marketplace/page.tsx]: On this button, remove the `size` property.

File: frontend/app/marketplace/page.tsx [hunk 2]
```diff
@@ -312,10 +313,17 @@ function HeaderComponent({ searchTerm, setSearchTerm }: HeaderComponentProps) {
...
+					<Wallet className="mr-2 h-4 w-4 transition-transform group-hover:scale-110" />
+					Connect Wallet
+				</Button>
+
+				<Button size="lg" className="group" onClick={() => router.push("/cart")}>
```
Comment (derianrddev) [inline frontend/app/marketplace/page.tsx]: On this other button, also remove the `size` property and the `mr-2`.

**Comment by derianrddev** [comment]
@SimplementeCao Good job! Just some minor comments. Please update the screenshot once you make the changes.

**Review by derianrddev — CHANGES_REQUESTED** [review]

**Comment by derianrddev** [comment]
@SimplementeCao Good job! Just some minor comments. Please update the screenshot once you make the changes.

**Comment by SimplementeCao** [comment]
@derianrddev Done! I have already modified the suggested changes and updated the images.

**Review by derianrddev — APPROVED** [review]

**Review by danielcdz — APPROVED** [review]
LGTM

**Pull request merged**